ports/esp32/esp32_rmt.c

```diff
@@ -96,7 +96,9 @@ STATIC mp_obj_t esp32_rmt_make_new(const mp_obj_type_t *type, size_t n_args, siz
     self->carrier_freq = carrier_freq;
     self->loop_en = false;
 
-    rmt_config_t config;
+    // start with an empty config, not what was on the stack
+
+    rmt_config_t config = {};
```

Comment:
Other parts of the code base use `= {0}` for this, so would be good to do that here as well, for consistency.

Also, IMO the comment is not needed, if you know C it's pretty clear what this is doing.

Reply:
Done.